webapp/view/ProductDetails.view.xml

```diff
@@ -0,0 +1,201 @@
+<mvc:View
+	xmlns="sap.m"
+	xmlns:mvc="sap.ui.core.mvc"
+    xmlns:u="sap.uxap"
+    xmlns:form="sap.ui.layout.form"
+	xmlns:core="sap.ui.core"
+    core:require="{formatMessage: 'sap/base/strings/formatMessage'}"
+	controllerName="products.app.controller.ProductDetails">
+
```

Comment:
Empty lines make it harder to read the code.
Please remove them. 
I suggest you to use vs code plugin 'xml tools', so you don't need format xml files manually